sensor_msgs/include/sensor_msgs/impl/point_cloud2_iterator.hpp

```diff
@@ -198,16 +181,6 @@ inline void PointCloud2Modifier::setPointCloud2Fields(int n_fields, ...)
   cloud_msg_.data.resize(cloud_msg_.height * cloud_msg_.row_step);
 }
 
-/**
- * @brief Function setting some fields in a PointCloud and adjusting the
- *        internals of the PointCloud2
- * @param n_fields the number of fields to add. The fields are given as
- *        strings: "xyz" (3 floats), "rgb" (3 uchar stacked in a float),
- *        "rgba" (4 uchar stacked in a float)
- * @return void
- *
- * WARNING: THIS FUNCTION DOES ADD ANY NECESSARY PADDING TRANSPARENTLY
- */
```

Comment:
Is this documentation block removed because it's an implementation file?

Reply:
Same answer as above.